tests/test_camera.py

```diff
@@ -1,8 +1,93 @@
+import pytest
+
 from ppb import BaseSprite
 from ppb import Vector
 from ppb.camera import Camera
+from ppb.events import SceneStarted
+
+
+def test_camera_sets_height():
+    """Guarantee that the height is initially set dynamically."""
+
+    cam = Camera()
+
+    # Going to keep the default frame of 10 game units wide.
+    assert cam.width == 10
+    # NotSet will be a flag so we have a sentinel value
+    assert cam.height is None
+
+    scene_started_event = SceneStarted(None)
+    scene_started_event.aspect_ratio = 2  # The width should be twice the height.
```

Comment:
This potential changes a lot? Setting it as part of the rendering process makes more sense?

So, my thought is, rendering can't be completely decoupled from cameras, and broadcasting the geometry doesn't seem super useful? Directly setting the geometry of each camera each frame still seems like the simplest, less weird way to do things.

Reply:
I was thinking having the renderer extend scene started.

But per my comment on discord: I can probably do this magic on heartbeat events to reduce the time delta between any camera getting instantiated or screen (or eventually viewport) size changing and the Camera being able to manage itself. And let the renderer just call a method or set a property on the camera directly.